lib/Runtime/Language/i386/StackFrame.cpp

```diff
@@ -51,9 +51,13 @@ X86StackFrame::InitializeByReturnAddress(void * returnAddress, ScriptContext* sc
 bool
 X86StackFrame::Next()
 {
-    this->addressOfCodeAddr = this->GetAddressOfReturnAddress();
-    this->codeAddr = this->GetReturnAddress();
-    this->frame = (void **)this->frame[0];
+    if (frame != nullptr)
+    {
```

Comment:
Only with nested enter_script, which I have removed.

---
In reply to: [114913205](https://github.com/Microsoft/ChakraCore/pull/2913#discussion_r114913205) [](ancestors = 114913205)